libs/ledger/tests/chaincode/token_contract_tests.cpp

```diff
@@ -270,12 +227,13 @@ TEST_F(TokenContractTests, CheckInitialBalance)
   EXPECT_EQ(balance, 0);
 }
 
+// TODO(HUT): this is disabled - by whom and why?
```

Comment:
Please raise an issue to track these please

Reply:
done